css-fonts-5/Overview.bs

```diff
@@ -178,53 +177,52 @@ the 'size-adjust' descriptor</h3>
-Name: ascent-override
-Value: normal | <<percentage [0,Infinity]>>
-For: @font-face
-Initial: normal
-</pre>
-
-<pre class='descdef'>
-Name: descent-override
-Value: normal | <<percentage [0,Infinity]>>
-For: @font-face
-Initial: normal
-</pre>
-
-<pre class='descdef'>
-Name: line-gap-override
-Value: normal | <<percentage [0,Infinity]>>
-For: @font-face
-Initial: normal
-</pre>
-
-The 'ascent-override', 'descent-override', and 'line-gap-override' descriptors define the
-<a spec="CSS-INLINE-3">ascent metric</a>, <a spec="CSS-INLINE-3">descent metric</a> and
-<a spec="CSS-INLINE-3">line gap metric</a> of the font, respectively.
-
-When the descriptor value is 'normal', the corresponding metric value is obtained as-if the
-descriptor was absent from the <code>@font-face</code> block.
-
-Note: User agents may gather data from different places from the font file as the metric values,
-which results in different text layouts.
-
-For the 'ascent-override', 'descent-override' and 'line-gap-override' descriptors, when the
-descriptor value is a percentage, the corresponding metric value is resolved as the
-given percentage multiplied by the used font size.
+	<pre class='descdef'>
+	Name: ascent-override
+	Value: normal | <<percentage [0,Infinity]>>
+	For: @font-face
+	Initial: normal
+	</pre>
 
-None of these descriptors affect the calculation of
-<a href="https://drafts.csswg.org/css-values-4/#font-relative-lengths">Font-relative Lengths</a>.
-None of these descriptors affect the behavior of any of the values of 'line-height' other than
-'normal'. The behavior of <code>line-height: normal</code> may be affected by all of these
-descriptors.
+	<pre class='descdef'>
+	Name: descent-override
+	Value: normal | <<percentage [0,Infinity]>>
+	For: @font-face
+	Initial: normal
+	</pre>
 
-Note: Using the 'ascent-override', 'descent-override', and 'line-gap-override' descriptors can
-achieve a similar effect as specifying a percentage for 'line-height' property.
+	<pre class='descdef'>
+	Name: line-gap-override
+	Value: normal | <<percentage [0,Infinity]>>
+	For: @font-face
+	Initial: normal
+	</pre>
 
-Note: 'ascent-override', 'descent-override', and 'line-gap-override'
-are applied per-font rather than per-element. Multiple fonts may be used
-within an individual element e.g. for characters not supported by the primary font.
+	The 'ascent-override', 'descent-override', and 'line-gap-override' descriptors specify the
+	<a spec="CSS-INLINE-3">ascent metric</a>,
+	<a spec="CSS-INLINE-3">descent metric</a>, and
+	<a spec="CSS-INLINE-3">line gap metric</a>
+	of the font, respectively.
+
+	<dl dfn-for="ascent-override!!descriptor,descent-override!!descriptor,line-gap-override!!descriptor" dfn-type=value>
+		<dt><dfn>normal</dfn>
+		<dd>
+			The corresponding metric value is obtained from the font as usual,
+			as if this descriptor were absent from the <code>@font-face</code> block.
+
+			Note: Since there are sources of such metrics in some font formats,
```

Comment:
Insert "multiple" before "sources", I think.

Reply:
Done! Thanks for catching that. :)